example_test.go

```diff
@@ -109,6 +110,7 @@ func ExampleParseWithClaims_validationOptions() {
 	}
 
 	token, err := jwt.ParseWithClaims(tokenString, &MyCustomClaims{}, func(token *jwt.Token) (interface{}, error) {
+    // []byte("AllYourBase") refers to the signing key that is used to sign the claim with
```

Comment:
not sure if we really need a comment here to be honest.

Reply:
We are somewhat "phasing" out those examples any way and want to build up our documentation website: https://golang-jwt.github.io/jwt/usage/parse/. Unfortunately, exactly the parsing aspect is not documented yet. I want to have that in the same style as https://golang-jwt.github.io/jwt/usage/create/, where each variable is explained in detail. If you want, you can of course contribute to the docs here: https://github.com/golang-jwt/jwt-docs/blob/main/docs/usage/parse.md

Reply:
Oh that looks neat, I will look into that, I have reverted the changes for now then 👍 .